<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_welcome">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Bine ai venit, Rares Babuta</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>0</y>
      <width>381</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QCommandLinkButton" name="commandLinkButton">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>0</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Profil</string>
     </property>
    </widget>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>190</y>
      <width>800</width>
      <height>305</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_main">
     <attribute name="title">
      <string>Rezultate</string>
     </attribute>
     <widget class="QTreeWidget" name="treeWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>230</height>
       </rect>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <column>
       <property name="text">
        <string>Nr.</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Titlu</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Autor</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Gen</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Subgen</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>An</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Rating</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>240</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_upload">
     <attribute name="title">
      <string>Upload</string>
     </attribute>
     <widget class="QLabel" name="label_titlu_carte">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>15</y>
        <width>100</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Titlu carte</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_nume_autor">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>75</y>
        <width>100</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Nume autor</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_prenume_autor">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>105</y>
        <width>100</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Preume autor</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_an_aparitie">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>135</y>
        <width>100</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>An aparitie</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_genuri">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>165</y>
        <width>50</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Genuri</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_subgenuri">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>217</y>
        <width>75</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Subgenuri</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="titlu_carte">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>270</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="nume_autor">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>70</y>
        <width>170</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="prenume_autor">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>100</y>
        <width>170</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="an_aparitie">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>130</y>
        <width>70</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="gen_plus">
      <property name="geometry">
       <rect>
        <x>75</x>
        <y>161</y>
        <width>23</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QPushButton" name="subgen_plus">
      <property name="geometry">
       <rect>
        <x>93</x>
        <y>215</y>
        <width>23</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="subgen1">
      <property name="geometry">
       <rect>
        <x>19</x>
        <y>240</y>
        <width>90</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_nume_autor_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>45</y>
        <width>100</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>ISBN</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="isbn">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>40</y>
        <width>270</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="descriere">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>40</y>
        <width>291</width>
        <height>55</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_descriere">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>10</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Descriere carte</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_coperta">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>108</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Incarcare coperta</string>
      </property>
     </widget>
     <widget class="QPushButton" name="browse_coperta">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>133</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_continut">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>173</y>
        <width>131</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Incarcare continut</string>
      </property>
     </widget>
     <widget class="QPushButton" name="browse_continut">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>203</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse</string>
      </property>
     </widget>
     <widget class="QLabel" name="status_coperta">
      <property name="geometry">
       <rect>
        <x>615</x>
        <y>135</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Nu ai selectat niciun fisier</string>
      </property>
     </widget>
     <widget class="QLabel" name="status_continut">
      <property name="geometry">
       <rect>
        <x>615</x>
        <y>205</y>
        <width>181</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Nu ai selectat niciun fisier</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="gen1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>187</y>
        <width>90</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="gen2">
      <property name="geometry">
       <rect>
        <x>115</x>
        <y>187</y>
        <width>90</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="gen3">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>187</y>
        <width>90</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="gen4">
      <property name="geometry">
       <rect>
        <x>305</x>
        <y>187</y>
        <width>90</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="gen_minus">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>161</y>
        <width>23</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QPushButton" name="subgen_minus">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>215</y>
        <width>23</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="subgen2">
      <property name="geometry">
       <rect>
        <x>115</x>
        <y>240</y>
        <width>90</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="subgen3">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>240</y>
        <width>90</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="subgen4">
      <property name="geometry">
       <rect>
        <x>305</x>
        <y>240</y>
        <width>90</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="upload_btn">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>242</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Upload</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>246</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>731</width>
      <height>141</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>81</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Nume autor</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Prenume autor</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>67</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Titlu</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="cautare_nume">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>8</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="cautare_prenume">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>38</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="cautare_titlu">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>68</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>10</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Gen</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="cautare_gen">
     <property name="geometry">
      <rect>
       <x>335</x>
       <y>8</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>40</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Subgen</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="cautare_subgen">
     <property name="geometry">
      <rect>
       <x>335</x>
       <y>38</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>70</y>
       <width>61</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>An</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="cautare_an">
     <property name="geometry">
      <rect>
       <x>335</x>
       <y>68</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>10</y>
       <width>41</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>ISBN</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="cautare_isbn">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>8</y>
       <width>211</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>50</y>
       <width>51</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Rating</string>
     </property>
    </widget>
    <widget class="QComboBox" name="cautare_rating">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>47</y>
       <width>85</width>
       <height>23</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="btn_cautare">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>106</y>
       <width>211</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Cautare</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>391</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>                                           Reads Profiler</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
